can/zynqmp-can/zynqmp-can.c

```diff
@@ -278,14 +278,16 @@ static int zynqmpCan_init(unsigned int can_id, int baudrate)
 		return -1;
 	}
 
-	/* Configure MIO pins */
+	/* Configure MIO pins if CAN is not routed via PL subsystem */
+#if (CAN_ROUTED_VIA_PL != 1)
 	if (zynqmpCan_configMio(info[can_id].rxPin) < 0 || zynqmpCan_configMio(info[can_id].txPin) < 0) {
 		fprintf(stderr, "zynqmp-can: failed to initialize MIO pins\n");
 		munmap((void *)can.base, _PAGE_SIZE);
 		resourceDestroy(can.cond);
 		resourceDestroy(can.lock);
 		return -1;
 	}
+#endif
```

Comment:
Instead of removing `static` I think you should be able to do instead:
```suggestion
#else
  (void)zynqmpCan_configMio;
#endif
```

Reply:
good point, fixed